iterative_verifier.go

```diff
@@ -244,6 +245,11 @@ func (v *IterativeVerifier) VerifyBeforeCutover() error {
 	return err
 }
 
+func (v *IterativeVerifier) SetApplicableTableSchemaCache(t TableSchemaCache) {
+	v.Tables = t.AsSlice()
+	v.TableSchemaCache = t
+}
```

Comment:
I still don't get why you're doing this instead of creating the verifier with the right fields

Reply:
As stated [earlier](https://github.com/Shopify/ghostferry/pull/89#discussion_r259859369), the loading order for the TableSchemaCache is somewhat important to this. I want to get rid of this too but it proved too difficult for one PR. There'll be a follow up PR that removes this method.